go.sum

```diff
@@ -526,6 +530,8 @@ go.uber.org/atomic v1.4.0/go.mod h1:gD2HeocX3+yG+ygLZcrzQJaqmWj9AIm7n08wl/qW/PE=
 go.uber.org/multierr v1.1.0/go.mod h1:wR5kodmAFQ0UK8QlbwjlSNy0Z68gJhDJUG5sjR94q/0=
 go.uber.org/multierr v1.2.0 h1:6I+W7f5VwC5SV9dNrZ3qXrDB9mD0dyGOi/ZJmYw03T4=
 go.uber.org/multierr v1.2.0/go.mod h1:wR5kodmAFQ0UK8QlbwjlSNy0Z68gJhDJUG5sjR94q/0=
+go.uber.org/ratelimit v0.1.0 h1:U2AruXqeTb4Eh9sYQSTrMhH8Cb7M0Ian2ibBOnBcnAw=
```

Comment:
does juju/ratelimit use uber's ratelimiter? 

I have a concern about the license of `juju/ratelimit`. it is licensed under `LGPL v3 License`.

Reply:
@yaron2 @amanbha any issues to use LGPL v3 licence module ?

Reply:
@youngbupark it doesn't use uber's ratelimiter (AFAIK that only does blocking rate limiting) but definitely worth checking the license obligations of LGPL v3.

Reply:
Normally, (at least Microsoft), production team normally excludes any LGPL/GPL licensed OSS. if it is required, it may require the review with LCA. So we need to be careful here.

Reply:
It’s a good catch - I will remove the reference to Uber/ratelimit anyway (I was testing it and left it in). If we can’t use juju ratelimit, then I could try alternatives like https://github.com/didip/tollbooth